packages/sensors_plus/sensors_plus/example/android/app/build.gradle

```diff
@@ -73,5 +65,4 @@ dependencies {
     testImplementation 'junit:junit:4.13.2'
     androidTestImplementation 'androidx.test:runner:1.5.2'
     androidTestImplementation 'androidx.test.espresso:espresso-core:3.5.1'
-    implementation "org.jetbrains.kotlin:kotlin-stdlib:$kotlin_version"
```

Comment:
It seems we don't really need to have Kotlin and its plugin in the example app for now, so removed it.